src/client/components/RoomListOverview/RoomListOverview.js

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
-import RoomList from '../RoomList/RoomList';
+import Channels from '../../containers/Channels/Channels';
```

Comment:
Components should not reference containers, only the other way round.

I suggest that you move 
`const { response: roomList, loading, error } = useFetch(`/api/channels`);` to `Overview.js` and simply pass roomList down to `<RoomListOverview />